react/features/base/color-scheme/defaultScheme.js

```diff
@@ -0,0 +1,23 @@
+// @flow
+
+import { ColorPalette, getRGBAFormat } from '../styles';
+
+/**
+ * The default color scheme of the application.
+ */
+export default {
```

Comment:
as discussed offline: let's make it so this is a nested object, indexed by feature name